20_beta_diversity.Rmd

```diff
@@ -282,10 +299,10 @@ them. The result shows how much each covariate affects beta diversity. The table
 below illustrates the relation between supervised and unsupervised ordination
 methods.
 
-|                           | supervised ordination  | unsupervised ordination  |
-|:-------------------------:|:----------------------:|:------------------------:|
-| Euclidean distance        | RDA                    | PCA                      |
-| non-Euclidean distance    | dbRDA                  | PCoA                     |
+|                          | supervised ordination  | unsupervised ordination  |
+|:------------------------:|:----------------------:|:------------------------:|
+| Euclidean distance       | RDA                    | PCA                      |
+| non-Euclidean distance   | dbRDA                  | PCoA                     |
 
```

Comment:
What is MDS? What is NMDS? What is UMAP? These info could be added

Reply:
Aren't they all subtypes of PCoA?

Reply:
PCA is a special case of PCoA (aka MDS) with Euclidean distances.

I am not sure if NMDS is exactly a special case of PCoA/MDS. It is closely related conceptually (focusing on order of distances rather than distances themselves) but saying that it is a special case would require more in-depth check. I don't  think MDS would directly reduce the NMDS but happy to be corrected if mistaken.

UMAP is a different, although related method. It is not a special case of any of the above.